specification/containerservice/resource-manager/Microsoft.ContainerService/aks/stable/2023-10-01/managedClusters.json

```diff
@@ -7401,6 +7411,19 @@
           "readOnly": true
         }
       }
+    },
+    "CapacityReservationGroupID": {
+      "type": "string",
+      "format": "arm-id",
+      "x-ms-arm-id-details": {
+        "allowedResources": [
+          {
+            "type": "Microsoft.Compute/CapacityReservationGroups"
+          }
+        ]
+      },
+      "title": "The fully qualified resource ID of the Capacity Reservation Group to provide virtual machines from a reserved group of Virtual Machines.",
+      "description": "This is of the form: '/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Compute/capacityreservationgroups/{capacityReservationGroupName}' Customers use it to create an agentpool with a specified CRG. For more information see [Capacity Reservation](https://learn.microsoft.com/en-us/azure/virtual-machines/capacity-reservation-overview)"
```

Comment:
[CI Swagger SpellCheck](https://github.com/Azure/azure-rest-api-specs/pull/26451/checks?check_run_id=18172624680) failed, please omit the form part in the description or create another PR to main branch directly and add the term `capacityreservationgroups` to `custom-words.txt`.

Reply:
A PR for custom-words.txt has been merged.